src/main/webapp/index.html

```diff
@@ -29,8 +29,11 @@
           <input id="origin-input" class="userInput" type="text" placeholder="Enter an origin location"/>
           <input id="destination-input" class="userInput" type="text" placeholder="Enter a destination location"/>
         </div>
+        <div id="waypoint-container">
+        </div>
         <div id="feature-buttons-container" class="flex-container">
-          <button class="feature-button" id="clean-air-button" alt="Find rest stops with clean air near this location" onClick="showCleanAirMarkers()">Clean air near here</button>
```

Comment:
This line needs to be kept, it starts the process where the markers are shown

Reply:
Because the comment interface is weird and you can't see the code below this line easily, I'll mention that the line is there in the current code, but without the onClick

Reply:
Yep! Sorry - this line got deleted in place of the other clean air button line of code. didnt catch the onclick at the end when i solved hthe merge conflict as the window to solve it on github was very narrow and you have to scroll to see the end of the line, and unforutnately i didnt catch it.

 it is fixed now! So now all three features are working together on the deployed version in this PR